.scrutinizer.yml

```diff
@@ -21,6 +21,7 @@ before_commands:
 tools:
     external_code_coverage:
         timeout: 3600
+        runs: 17
```

Comment:
If the syntax allows, could you leave a comment about how the expected number of runs is determined? It'll have to be synchronized with the changes in `.travis.yml` and ContinuousPHP configuration. AppVeyor probably doesn't count since it doesn't cover any additional DB platforms.

Reply:
YAML comment is used below so it should be ok. Actually forgot to count ContinuousPHP in. :) We can add AppVeyor too for the older (2008, 2012) platforms.